common/src/test/java/com/revenuecat/purchases/common/verification/SigningManagerTest.kt

```diff
@@ -146,35 +153,31 @@ class SigningManagerTest {
 
     @Test
     fun `verifyResponse returns success if verifier returns success for given parameters`() {
-        every { verifier.verify(any(), any()) } returns true
+        every { intermediateKeyVerifier.verify(any(), any()) } returns true
         val verificationResult = callVerifyResponse(informationalSigningManager)
         assertThat(verificationResult).isEqualTo(VerificationResult.VERIFIED)
     }
 
     @Test
     fun `verifyResponse returns error if verifier returns success for given parameters`() {
-        every { verifier.verify(any(), any()) } returns false
+        every { intermediateKeyVerifier.verify(any(), any()) } returns false
         val verificationResult = callVerifyResponse(informationalSigningManager)
         assertThat(verificationResult).isEqualTo(VerificationResult.FAILED)
     }
 
     @Test
     fun `verifyResponse with real data verifies correctly`() {
-        val signingManager = SigningManager(
-            SignatureVerificationMode.Informational(DefaultSignatureVerifier()),
-            appConfig
-        )
+        val rootVerifier = DefaultSignatureVerifier("yg2wZGAr8Af+Unt9RImQDbL7qA81txk+ga0I+ylmcyo=")
```

Comment:
Note that now we aren't using the real root key but a new test key. It was difficult to test using the real key since the current responses may become outdated once the intermediate key expires. Still plan to add some integration tests that test this against the backend.

Reply:
Hmm how so? The expiration is also encoded in the signature.

Reply:
Right but we don't have the real root private key in order to create a signed intermediate key with an expiration date far in the future. If not, we would need to be changing the signature every time the intermediate key expires I think? I guess maybe we could get a signature that expires far into the future but not sure if that would have any security concerns... Probably not but I will think about that a bit more

Reply:
Oooh I see. I got around that by creating fake private and public root keys in some of these tests.

Reply:
Right, that's what I'm doing here. I basically used test private/public keys to generate signatures with different values that I need. But we are not testing using the real production key as we were before... But I think it's fine if that's covered by integration tests that actually hit the backend.